core/pkg/telemetry/metrics.go

```diff
@@ -131,27 +131,27 @@ func NewOTelRecorder(exporter msdk.Reader, resource *resource.Resource, serviceN
 	// we can ignore errors from OpenTelemetry since they could occur if we select the wrong aggregator
 	hduration, _ := meter.Float64Histogram(
 		httpRequestDurationMetric,
-		metric.WithDescription("The latency of the HTTP requests"),
+		metric.WithDescription("Measures the duration of inbound HTTP requests."),
```

Comment:
no change these are exported from the semconv package, is there?

Reply:
Seems not 🤔  there's only attribute key value exposed